### PR TITLE
0207 hospital beds

### LINKED ISSUE #207: Visualize hospital bed capacity

### PR DESCRIPTION
**Pull request by jaryaman**

### CODE REVIEW

File: work/covidvu/pipeline/vuhospitals.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import os
+from os.path import join
+
+from os.path import join
+from time import sleep
+
+# *** constants ***
+STATE_CODES_PATH = join(os.getcwd(), 'stateCodesUS.csv')
+SITE_DATA               = './site-data'
```
Comment (pr3d4t0r) [inline work/covidvu/pipeline/vuhospitals.py]: This is defined in one of the modules, I believe covidvu.pipeline.vujson - let's fix it later.  I'll merge/approve this.

File: work/covidvu/pipeline/vuhospitals.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+OUT_FILE_NAME = 'hospital_bed_count.json'
```
Comment (pr3d4t0r) [inline work/covidvu/pipeline/vuhospitals.py]: Dashes for file names, not underscores (underscores are considered alphanum characters, dashes aren't - important when looking at processing huge numbers of files in the file system or an S3 bucket.

File: work/covidvu/pipeline/vuhospitals.py (hunk 1 above)
Comment (jaryaman) [reply work/covidvu/pipeline/vuhospitals.py]: Yep, you're right.

File: work/covidvu/pipeline/vuhospitals.py (hunk 2 above)
Comment (jaryaman) [reply work/covidvu/pipeline/vuhospitals.py]: Thanks for this! Great tip.

**Pull request merged**